website/blog/2022-07-10-presto-parquet-column-encryption.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Presto Parquet Column Encryption
+author: Xinli Shang
+authorURL: https://www.linkedin.com/in/xinli-shang-2479863/
+---
+**Uber:** Xinli Shang
+
+## Introduction
+Apache Parquet modular encryption provides encryption at-rest and in-transit at finer-grained. In big data world, data analytic tables are usually very wide with hundreds of columns, while only a small number of columns need to be protected. So the finer-grained access control is a better fit than coarse-grained one like table level access control.
+
+In addition, data access restrictions, retention, and encryption at rest are fundamental security controls. Column encryption with access control at the encryption key can solve all three problems with one unified solution as discussed in another blog [One Stone, Three Birds: Finer-Grained Encryption @ Apache Parquet™](https://eng.uber.com/one-stone-three-birds-finer-grained-encryption-apache-parquet).
```

Comment:
we have mix use of "at-rest" and "at rest". Might be good to unify them.

Reply:
Good point. Fixed it now